### DISCUSSION TITLE
Question About Asking for Extensions

### DISCUSSION THREAD

**Discussion by sep18012** [cooperrc/computational-mechanics#89] [General]
Prof Cooper had spoke about asking for extensions in class today and I understand that we need to provide when we can submit our work and what is giving us trouble. But, where exactly should we be letting the professor know about this? Would email be best? Also, should we be including the TA's in this email?

**Comment by WRea16** [comment]
I am also having trouble with the first homework, specifically problem 3. I emailed, but have not heard back. If at all there can be an extension that would be greatly appreciated as I have been struggling for many hours with just problem 3.

**Comment by cooperrc** [comment]
My UConn email is great to let me know you're running late on submission. Thanks!
